run.sh

```diff
@@ -59,12 +65,13 @@ function run {
         -e "RUM_URL=$RUM_URL" \
         -e "ENABLE_HTTPS=$ENABLE_HTTPS" \
         -e "ANOMALY_EMAIL=$ANOMALY_EMAIL" \
+        -e "COSCALE_VERSION=$IMAGE_VERSION" \
         --name coscale_$SERVICE coscale/$SERVICE:$IMAGE_VERSION
 }
 
 # Run the data services
 for SERVICE in $DATA_SERVICES; do
-    if [ "$NAME" == "all" ] || [ "$NAME" == "data" ] || [ "$NAME" == "$SERVICE" ]; then
+    if [ "$NAME" == "all" ] || [ "$NAME" == "$SERVICE" ]; then
```

Comment:
I would like to keep the "data" option, sometimes it is handy to start the data services first without the CoScale services.